tests/integration/test_fim/test_basic_usage/test_basic_usage_deferred_delete_folder.py

```diff
@@ -0,0 +1,128 @@
+# Copyright (C) 2015-2020, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+import os
+from subprocess import Popen, PIPE, DEVNULL
+import re
+import json
+from json import JSONDecodeError
+
+import pytest
+
+from wazuh_testing import global_parameters
+from wazuh_testing.fim import LOG_FILE_PATH, generate_params, create_file, REGULAR, callback_detect_event
+from wazuh_testing.tools import PREFIX
+from wazuh_testing.tools.configuration import load_wazuh_configurations, check_apply_test
+from wazuh_testing.tools.monitoring import FileMonitor
+
+# Marks
+
+pytestmark = [pytest.mark.win32, pytest.mark.tier(level=0)]
+
+# variables
+
+wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
+test_directories = [os.path.join(PREFIX, 'testdir1'), os.path.join(PREFIX, 'testdir2')]
+directory_str = ','.join(test_directories)
+for direc in list(test_directories):
+    test_directories.append(os.path.join(direc, 'subdir'))
+test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+configurations_path = os.path.join(test_data_path, 'wazuh_conf.yaml')
+testdir1, testdir2 = test_directories[2:]
+
+# configurations
+
+conf_params = {'TEST_DIRECTORIES': directory_str, 'MODULE_NAME': __name__}
+p, m = generate_params(extra_params=conf_params, modes=['whodata'])
+configurations = load_wazuh_configurations(
+    configurations_path, __name__, params=p, metadata=m)
+
+
+# callback
+def callback_detect_delete_event(line):
+    msg = r'.*Sending FIM event: (.+)$'
+    match = re.match(msg, line)
+
+    try:
+        event = json.loads(match.group(1))
+        if (event['type'] == 'event' and
+            event['data']['type'] == 'deleted' and
+            'process_name' not in event['data']['audit']):

```

Comment:
Add one tabulation to this line, it shouldn't be at the same indent as the next logical line.